docs/streams/kafka_on_gdn/index.md

```diff
@@ -0,0 +1,16 @@
+---
+sidebar_position: 150
+title: Kafka on GDN
+---
+
+Kafka on GDN brings the native Apache Kafka protocol support to c8streams by introducing a Kafka protocol handler on c8streams brokers. By adding the Kafka on GDN protocol handler to your existing c8streams cluster, you can migrate your existing Kafka applications and services to c8streams without modifying the code. This enables Kafka applications to leverage c8streams’s powerful features
```

Comment:
@chandrashekhar-macrometa - `c8streams` is for internal use only. Please replace it by `GDN streams`.

Reply:
Agreed. We have been directed to avoid using "c8" wherever possible.